apps/web/components/rating/RatingDelta.tsx

```diff
@@ -0,0 +1,91 @@
+'use client';
+
+import { cn } from '@/lib/utils';
+import { TrendingUp, TrendingDown, Minus } from 'lucide-react';
+import { useMemo } from 'react';
+
+const RATING_PRECISION = {
+  DELTA: 1,
+  COMPARISON: 10,
+} as const;
+
+const RATING_THRESHOLDS = {
+  HIGH_GAIN: 10,
+  HIGH_LOSS: -10,
+  EXTREME: 30,
+  MODERATE: 15,
+} as const;
```

Comment:
Shouldn't there just be `HIGH` with a positive value? Maybe high could be 20, moderate 40, and extreme 60? High loss would just be `-HIGH_GAIN`